src/components/canvas-tools/geometry-tool.tsx

```diff
@@ -600,7 +600,7 @@ class GeometryToolComponentImpl extends BaseComponent<IProps, IState> {
     this.dragSelectedPoints(evt, usrDiff);
 
     // only create a change object if there's actually a change
-    if ((usrDiff[0] !== 0) || (usrDiff[1] !== 0)) {
+    if (usrDiff[1] || usrDiff[2]) {
```

Comment:
Good catch. It seems like an easy bug that x is at index 1 and y is at 2. What's at 0? Would it help if coords were always turned into an object with named keys whenever we were trying to use it like this?

Reply:
Internally JSXGraph uses normalized coordinates of the form [1, x, y]. I did a search for `[0]` to see if there were other problematic uses and didn't find any. What I've used in some places and could use here is something like:
```typescript
const [ , xDiff, yDiff] = usrDiff;
...
```